internal/kernel/scheduler/scheduler.go

```diff
@@ -88,7 +90,11 @@ func (s *Scheduler) Tick(t int64, batchSize int) {
 						}
 					}
 				},
-			})
+			}
+
+			util.Assert(sqe.Callback != nil, "sqe is nil")
```

Comment:
can we add two additional assertions: onw making sure metadata is not nil and same for submission?